core/templates/domain/collection/collection-node.model.ts

```diff
@@ -41,6 +38,19 @@ export class CollectionNode {
       collectionNodeBackendObject.exploration_summary);
   }
 
+  static create(
+      collectionNodeBackendObject: CollectionNodeBackendDict):
+      CollectionNode {
+    return new CollectionNode(collectionNodeBackendObject);
+  }
+
+  static createFromExplorationId(explorationId: string): CollectionNode {
+    return this.create({
+      exploration_id: explorationId,
+      exploration_summary: null
+    });
```

Comment:
static functions don't have access to the `this` keyword. This refers to the instance of the model but static functions are bound to the class and not the instance. So this will become:
```suggestion
    return CollectionNode.create({
      exploration_id: explorationId,
      exploration_summary: null
    });
```

Reply:
Done here and elsewhere!